### ISSUE TITLE
REFL001 should warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/ReflectionAnalyzers#125]
```cs
public class Foo
{
    public Foo(int i)
    {
        var info = typeof(Foo).GetConstructor(new[] { typeof(int) });
        var value = ↓info.Invoke(new object[] { 1 });
    }
}
```
